interfaces/builtin/docker_support.go

```diff
@@ -117,6 +117,9 @@ pivot_root,
 /etc/apparmor.d/cache/ r,
 /etc/apparmor.d/cache/.features r,
 /etc/apparmor.d/{,cache/}docker* rw,
+/var/cache/apparmor/{,*/} r,
```

Comment:
wonder if this list and the preexsting ones merit a comment about which apparmor versions/variants they apply to

Reply:
Done